Description: Fix 'messge' typo, replace with 'message'.
Author: Bas Couwenberg <sebastic@debian.org>
Forwarded: not-needed

--- a/frmts/grib/degrib18/g2clib-1.0.4/g2_addfield.c
+++ b/frmts/grib/degrib18/g2clib-1.0.4/g2_addfield.c
@@ -139,7 +139,7 @@ g2int g2_addfield(unsigned char *cgrib,g
 //
       if ( cgrib[0]!=G || cgrib[1]!=R || cgrib[2]!=I || cgrib[3]!=B ) {
         printf("g2_addfield: GRIB not found in given message.\n");
-        printf("g2_addfield: Call to routine g2_create required to initialize GRIB messge.\n");
+        printf("g2_addfield: Call to routine g2_create required to initialize GRIB message.\n");
         ierr=-1;
         return(ierr);
       }
--- a/frmts/grib/degrib18/g2clib-1.0.4/g2_addgrid.c
+++ b/frmts/grib/degrib18/g2clib-1.0.4/g2_addgrid.c
@@ -89,7 +89,7 @@ g2int g2_addgrid(unsigned char *cgrib,g2
 //
       if ( cgrib[0]!=G || cgrib[1]!=R || cgrib[2]!=I || cgrib[3]!=B ) {
         printf("g2_addgrid: GRIB not found in given message.\n");
-        printf("g2_addgrid: Call to routine gribcreate required to initialize GRIB messge.\n");
+        printf("g2_addgrid: Call to routine gribcreate required to initialize GRIB message.\n");
         ierr=-1;
         return(ierr);
       }
--- a/frmts/grib/degrib18/g2clib-1.0.4/g2_addlocal.c
+++ b/frmts/grib/degrib18/g2clib-1.0.4/g2_addlocal.c
@@ -67,7 +67,7 @@ g2int g2_addlocal(unsigned char *cgrib,u
 //
       if ( cgrib[0]!=G || cgrib[1]!=R || cgrib[2]!=I || cgrib[3]!=B ) {
         printf("g2_addlocal: GRIB not found in given message.\n");
-        printf("g2_addlocal: Call to routine g2_create required to initialize GRIB messge.\n");
+        printf("g2_addlocal: Call to routine g2_create required to initialize GRIB message.\n");
         ierr=-1;
         return(ierr);
       }
