Description: Fix 'postion' typo, replace with 'position'.
Author: Bas Couwenberg <sebastic@debian.org>
Forwarded: not-needed

--- a/apps/ogr_utilities.dox
+++ b/apps/ogr_utilities.dox
@@ -566,7 +566,7 @@ format. Also some information writed to
 <dt> <b>-of</b><em>field_name</em>:</dt><dd> The field name for storing the 
     uniq values of input lines</dd>
 <dt> <b>-s</b><em>step</em>:</dt><dd> The part size in linear units</dd>
-<dt> <b>-get_pos</b>:</dt><dd> Return linear referenced postion for input X, Y</dd>
+<dt> <b>-get_pos</b>:</dt><dd> Return linear referenced position for input X, Y</dd>
 <dt> <b>-x</b><em>long</em>:</dt><dd> Input X coordinate</dd>
 <dt> <b>-y</b><em>lat</em>:</dt><dd> Input Y coordinate</dd>
 <dt> <b>-get_coord</b>:</dt><dd> Return point on path for input linear distance</dd>
--- a/apps/ogrlineref.cpp
+++ b/apps/ogrlineref.cpp
@@ -114,7 +114,7 @@ static void Usage(const char* pszAdditio
         " -lf field_name: Field name for uniq paths in layer (optional)\n"
         " -p src_repers_datasource_name: Datasource of repers name\n"
         " -pn layer_name: Layer name in datasource (optional)\n"
-        " -pm pos_field_name: Line postion field name\n"
+        " -pm pos_field_name: Line position field name\n"
         " -pf field_name: Field name for correspondence repers of separate paths in layer (optional)\n"
         " -r src_parts_datasource_name: Parts datasource name\n"
         " -rn layer_name: Layer name in datasource (optional)\n"
--- a/frmts/hdf4/hdf-eos/EHapi.c
+++ b/frmts/hdf4/hdf-eos/EHapi.c
@@ -1702,7 +1702,7 @@ EHinsertmeta(int32 sdInterfaceID, char *
     char           *metaArr[2];	/* Array of metadata positions */
     char           *colon;	/* Colon position */
     char           *colon2;	/* 2nd colon position */
-    char           *slash;	/* Slash postion */
+    char           *slash;	/* Slash position */
     char           *utlstr;	/* Utility string */
     char           *utlstr2;	/* Utility string 2 */
 
@@ -2301,8 +2301,8 @@ EHinsertmeta(int32 sdInterfaceID, char *
     /* --------------------------------------- */
     seglen = strlen(utlstr);
 
-    /* Get offset of entry postion within existing metadata */
-    /* ---------------------------------------------------- */
+    /* Get offset of entry position within existing metadata */
+    /* ----------------------------------------------------- */
     offset = metaptr - metabuf;
 
 
--- a/frmts/netcdf/netcdfdataset.cpp
+++ b/frmts/netcdf/netcdfdataset.cpp
@@ -4378,7 +4378,7 @@ GDALDataset *netCDFDataset::Open( GDALOp
     int          nVarID=-1;
 
     int          nTmpFormat=NCDF_FORMAT_NONE;
-    int          *panBandDimPos=NULL;         // X, Y, Z postion in array
+    int          *panBandDimPos=NULL;         // X, Y, Z position in array
     int          *panBandZLev=NULL;
     int          *paDimIds=NULL;
     size_t       xdim, ydim;
@@ -5325,7 +5325,7 @@ netCDFDataset::CreateCopy( const char *
     int status = NC_NOERR;
 
     int nDim = 2;
-    int          *panBandDimPos=NULL;         // X, Y, Z postion in array
+    int          *panBandDimPos=NULL;         // X, Y, Z position in array
     int          *panBandZLev=NULL;
     int          *panDimIds=NULL;
     int          *panDimVarIds=NULL;
--- a/man/man1/ogrlineref.1
+++ b/man/man1/ogrlineref.1
@@ -75,7 +75,7 @@ The field name for storing the uniq valu
 .IP "\fB\fB-s\fP\fIstep\fP:\fP" 1c
 The part size in linear units 
 .IP "\fB\fB-get_pos\fP:\fP" 1c
-Return linear referenced postion for input X, Y 
+Return linear referenced position for input X, Y 
 .IP "\fB\fB-x\fP\fIlong\fP:\fP" 1c
 Input X coordinate 
 .IP "\fB\fB-y\fP\fIlat\fP:\fP" 1c
