Description: Fix 'responsiblity' typo, replace with 'responsibility'.
Author: Bas Couwenberg <sebastic@debian.org>

--- a/ogr/ogr_srsnode.cpp
+++ b/ogr/ogr_srsnode.cpp
@@ -342,7 +342,7 @@ void OGR_SRSNode::SetValue( const char *
 /**
  * Make a duplicate of this node, and it's children.
  *
- * @return a new node tree, which becomes the responsiblity of the caller.
+ * @return a new node tree, which becomes the responsibility of the caller.
  */
 
 OGR_SRSNode *OGR_SRSNode::Clone() const
--- a/ogr/ogrsf_frmts/ogrsf_frmts.dox
+++ b/ogr/ogrsf_frmts/ogrsf_frmts.dox
@@ -703,7 +703,7 @@ by the OGRSFDriverManager.
  \brief Fetch the next available feature from this layer.
 
  The returned feature
- becomes the responsiblity of the caller to delete with OGRFeature::DestroyFeature(). It is critical that
+ becomes the responsibility of the caller to delete with OGRFeature::DestroyFeature(). It is critical that
  all features associated with an OGRLayer (more specifically an 
  OGRFeatureDefn) be deleted before that layer/datasource is deleted.
 
@@ -735,7 +735,7 @@ by the OGRSFDriverManager.
  \brief Fetch the next available feature from this layer.
 
  The returned feature
- becomes the responsiblity of the caller to delete with OGR_F_Destroy().  It is critical that
+ becomes the responsibility of the caller to delete with OGR_F_Destroy().  It is critical that
  all features associated with an OGRLayer (more specifically an 
  OGRFeatureDefn) be deleted before that layer/datasource is deleted.
 
--- a/swig/include/python/docs/ogr_layer_docs.i
+++ b/swig/include/python/docs/ogr_layer_docs.i
@@ -181,7 +181,7 @@ OGR_L_GetNextFeature(OGRLayerH hLayer)
 
 Fetch the next available feature from this layer.
 
-The returned feature becomes the responsiblity of the caller to delete
+The returned feature becomes the responsibility of the caller to delete
 with OGR_F_Destroy(). It is critical that all features associated with
 an OGRLayer (more specifically an OGRFeatureDefn) be deleted before
 that layer/datasource is deleted.
--- a/swig/python/extensions/ogr_wrap.cpp
+++ b/swig/python/extensions/ogr_wrap.cpp
@@ -24357,7 +24357,7 @@ static PyMethodDef SwigMethods[] = {
 		"\n"
 		"Fetch the next available feature from this layer.\n"
 		"\n"
-		"The returned feature becomes the responsiblity of the caller to delete\n"
+		"The returned feature becomes the responsibility of the caller to delete\n"
 		"with OGR_F_Destroy(). It is critical that all features associated with\n"
 		"an OGRLayer (more specifically an OGRFeatureDefn) be deleted before\n"
 		"that layer/datasource is deleted.\n"
--- a/swig/python/osgeo/ogr.py
+++ b/swig/python/osgeo/ogr.py
@@ -1380,7 +1380,7 @@ class Layer(MajorObject):
 
         Fetch the next available feature from this layer.
 
-        The returned feature becomes the responsiblity of the caller to delete
+        The returned feature becomes the responsibility of the caller to delete
         with OGR_F_Destroy(). It is critical that all features associated with
         an OGRLayer (more specifically an OGRFeatureDefn) be deleted before
         that layer/datasource is deleted.
