Description: Fix 'retrive' typo, replace with 'retrieve'.
Author: Bas Couwenberg <sebastic@debian.org>

--- a/frmts/raw/ehdrdataset.cpp
+++ b/frmts/raw/ehdrdataset.cpp
@@ -1600,7 +1600,7 @@ GDALDataset *EHdrDataset::Open( GDALOpen
             }
             else
             {
-                CPLError( CE_Warning, CPLE_NotSupported, "Cannot retrive projection from IMAGE.REP");
+                CPLError( CE_Warning, CPLE_NotSupported, "Cannot retrieve projection from IMAGE.REP");
             }
         }
     }
