Description: Fix spelling errors.
 * Invalide -> Invalid
 * Altitide -> Altitude
Author: Bas Couwenberg <sebastic@debian.org>

--- a/ogr/ogrsf_frmts/kml/ogrkmldatasource.cpp
+++ b/ogr/ogrsf_frmts/kml/ogrkmldatasource.cpp
@@ -296,7 +296,7 @@ int OGRKMLDataSource::Create( const char
             CPLFree( pszAltitudeMode_ );
             pszAltitudeMode_ = NULL;
             CPLError( CE_Warning, CPLE_AppDefined,
-                      "Invalide AltitideMode specified, ignoring" );
+                      "Invalid AltitudeMode specified, ignoring" );
         }
     }
     else
