Description: Fix 'tranform' typo, replace with 'transform'.
Author: Bas Couwenberg <sebastic@debian.org>

--- a/NEWS
+++ b/NEWS
@@ -466,7 +466,7 @@ Algorithms:
  * Warper: fix integer overflow when reprojecting into an area with (part of) bounds completely outside of the source projection (#5789)
  * Warper: add min,max,med,q1 and q3 resampling algorithms (#5868)
  * Warper: add a SRC_COORD_PRECISION warping option to help getting more reproducable output when -wm parameter changes (#5925)
- * Warper: fix failure in GDALSuggestedWarpOut2() when top-left and bottom-right corners tranform to the same point (#5980)
+ * Warper: fix failure in GDALSuggestedWarpOut2() when top-left and bottom-right corners transform to the same point (#5980)
  * GDALReprojectImage(): takes into account nodata values set on destination dataset
  * Median cut and dithering: optimizations and enhancements to deal with 8-bit precision (only if using internal interface for now)
  * rasterfill: add option to specify driver to use for temporary files
--- a/frmts/grib/degrib18/degrib/tdlpack.cpp
+++ b/frmts/grib/degrib18/degrib/tdlpack.cpp
@@ -48,9 +48,9 @@ TDLP_TableType TDLP_V_Table[4] = {
 };
 
 TDLP_TableType TDLP_T_Table[3] = {
-   /* 0 */ {0, "No nolinear tranform"},
+   /* 0 */ {0, "No nolinear transform"},
    /* 1 */ {1, "Square transform"},
-   /* 2 */ {2, "Square root tranform"},
+   /* 2 */ {2, "Square root transform"},
 };
 
 TDLP_TableType TDLP_Oper_Table[9] = {
--- a/frmts/ingr/IngrTypes.cpp
+++ b/frmts/ingr/IngrTypes.cpp
@@ -301,9 +301,9 @@ void CPL_STDCALL INGR_GetTransMatrix( IN
         return;
     }
 
-    // -------------------------------------------------------------
-    // Calculate Concatened Tranformation Matrix based on Orientation
-    // -------------------------------------------------------------
+    // ---------------------------------------------------------------
+    // Calculate Concatened Transformation Matrix based on Orientation
+    // ---------------------------------------------------------------
 
     double adfConcat[16];
    
--- a/frmts/netcdf/netcdfdataset.cpp
+++ b/frmts/netcdf/netcdfdataset.cpp
@@ -3553,7 +3553,7 @@ CPLErr netCDFDataset::AddProjectionVars(
                          adfGeoTransform[i] );
                 strcat( szGeoTransform, szTemp );
             }
-            CPLDebug( "GDAL_netCDF", "szGeoTranform = %s", szGeoTransform );
+            CPLDebug( "GDAL_netCDF", "szGeoTransform = %s", szGeoTransform );
             
             // if ( strlen(pszProj4Defn) > 0 ) {
             //     nc_put_att_text( cdfid, NCDFVarID, "proj4",
--- a/frmts/png/libpng/png.h
+++ b/frmts/png/libpng/png.h
@@ -1399,7 +1399,7 @@ struct png_struct_def
 #endif
 
 #if defined(PNG_READ_SHIFT_SUPPORTED) || defined(PNG_WRITE_SHIFT_SUPPORTED)
-   png_color_8 shift PNG_DEPSTRUCT;         /* shift for significant bit tranformation */
+   png_color_8 shift PNG_DEPSTRUCT;         /* shift for significant bit transformation */
 #endif
 
 #if defined(PNG_tRNS_SUPPORTED) || defined(PNG_READ_BACKGROUND_SUPPORTED) \
